### ISSUE TITLE
Error occurred when executing DetailerForEachPipeForAnimateDiff:

### ISSUE THREAD

**Issue by WebKing2024** [ltdrdata/ComfyUI-Impact-Pack#628]
I'm not sure if this error is caused by Impack-Pack. Please see the complete error message 
![2024-06-06_175036](https://github.com/ltdrdata/ComfyUI-Impact-Pack/assets/160806472/a00b0157-cc51-4a5e-bb61-770cf6e00334)
:

Input and output must have the same number of spatial dimensions, but got input with spatial dimensions of [23, 680, 512] and output size of (816, 614). Please provide input tensor in (N, C, d1, d2, ...,dK) format and output size in (o1, o2, ...,oK) format.

File "C:\ComfyUI-aki-v1.2\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "C:\ComfyUI-aki-v1.2\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "C:\ComfyUI-aki-v1.2\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "C:\ComfyUI-aki-v1.2\custom_nodes\ComfyUI-Impact-Pack\modules\impact\animatediff_nodes.py", line 162, in doit
enhanced_seg, cnet_images = SEGSDetailerForAnimateDiff().do_detail(image_frames, single_seg, guide_size, guide_size_for, max_size, seed, steps, cfg, sampler_name, scheduler,
File "C:\ComfyUI-aki-v1.2\custom_nodes\ComfyUI-Impact-Pack\modules\impact\animatediff_nodes.py", line 50, in do_detail
segs = core.segs_scale_match(segs, image_frames.shape)
File "C:\ComfyUI-aki-v1.2\custom_nodes\ComfyUI-Impact-Pack\modules\impact\core.py", line 828, in segs_scale_match
cropped_mask = torch.nn.functional.interpolate(cropped_mask.unsqueeze(0).unsqueeze(0), size=(new_h, new_w), mode='bilinear', align_corners=False)
File "C:\ComfyUI-aki-v1.2\python\lib\site-packages\torch\nn\functional.py", line 3916, in interpolate
raise ValueError(

**Comment by WebKing2024** [comment]
Could any big guy help me solve this issues? Thank u very much!

**Comment by ltdrdata** [comment]
What is your workflow? I need to check the passed mask in SEGS.

**Comment by WebKing2024** [comment]
> What is your workflow? I need to check the passed mask in SEGS.

[workflow(29)1.json](https://github.com/user-attachments/files/15688620/workflow.29.1.json)

This is my workflow with the issue and i update Impact-Pack vision id: 6b9d862 nothing changed. hope u have a nice day!

**Comment by WebKing2024** [comment]
> What is your workflow? I need to check the passed mask in SEGS.

[workflow(29)1.json](https://github.com/user-attachments/files/15688620/workflow.29.1.json)

This is my workflow with the issue and i update Impact-Pack vision id: 6b9d862 nothing changed. hope u have a nice day!

**Comment by ltdrdata** [comment]
This is fixed.
